packages/flutter_tools/gradle/flutter.gradle

```diff
@@ -633,11 +702,8 @@ abstract class BaseFlutterTask extends DefaultTask {
                 if (extraGenSnapshotOptions != null) {
                     args "--extra-gen-snapshot-options", "${extraGenSnapshotOptions}"
                 }
-                if (buildSharedLibrary) {
-                    args "--build-shared-library"
-                }
                 if (targetPlatform == null) {
```

Comment:
Can this ever be null anymore? (is the parameter truly optional?)

Reply:
👍